Source/JavaScriptCore/API/JSValue.h

```diff
@@ -171,6 +171,44 @@ NS_CLASS_AVAILABLE(10_9, 7_0)
 */
 + (JSValue *)valueWithNewSymbolFromDescription:(NSString *)description inContext:(JSContext *)context JSC_API_AVAILABLE(macos(10.15), ios(13.0));
 
+/*!
+@method
+@abstract Create a new, unique, bigint object.
+@param string The string representation of the bigint object being created.
+@param context The JSContext to which the resulting JSValue belongs.
+@result The JSValue representing a unique JavaScript value with type bigint.
+@discussion This is equivalent to calling the `BigInt` constructor from JavaScript with a string argument.
+*/
++ (JSValue *)valueWithNewBigIntFromString:(NSString *)string inContext:(JSContext *)context JSC_API_AVAILABLE(macos(JSC_MAC_TBA), ios(JSC_IOS_TBA));
```

Comment:
I understand we need to add `_Null_unspecified` for other existing API. But still seems a bit weird since only the new APIs have specific whether it's nullable related annotations.